cb-sol-cli/cmd/deploy.js

```diff
@@ -24,6 +27,12 @@ const deployCmd = new Command("deploy")
     .option('--erc721Symbol <symbol>', 'Name for the erc721 contract', "")
     .option('--erc721Name <name>', 'Symbol for the erc721 contract', "")
     .option('--erc721BaseUri <uri>', 'Base URI for the erc721 contract', "")
+    .option('--erc1155', 'Deploy erc1155 contract')
+    .option('--erc1155Symbol <symbol>', 'Name for the erc1155 contract', "")
+    .option('--erc1155Name <name>', 'Symbol for the erc1155 contract', "")
+    .option('--erc1155BaseUri <uri>', 'Base URI for the erc1155 contract', "")
+    .option('--rollupExample', 'Deploy rollupExample contract')
+    .option('--rollupResourceID <resourceID>', 'resource for RollupExample contract', "")
```

Comment:
For consistency, we can change `rollupResourceID` -> `rollupResourceId` regarding `--chainId`
https://github.com/alt-research/altbridge-deploy/blob/dad751d345c7c7208becb84c48319e274e0b460a/cb-sol-cli/cmd/deploy.js#L9